packages/ethereum-contracts/contracts/utils/AllowList.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
Can also make a simple usage of OZ's roles contracts. Regardless of how simple it is, not having to rebuild the would would be my starting point.